### DISCUSSION TITLE
Update helm charts with seq:2023.2

### DISCUSSION THREAD

**Discussion by ThomasErvik** [datalust/seq-tickets#1864] [Help]
Hi,
When will the helm chart be updated to support SEQ:2023.2 ? 
-Thomas

**Comment by KodrAus** [comment]
Hi @ThomasErvik 👋 

We’ve decoupled the release process for the helm chart from Seq itself, so will issue another release of it when there are new features in the chart itself.

You can use the [`image.tag` property](https://github.com/datalust/helm.datalust.co/blob/main/samples/seq/config.yaml#LL2C8-L2C8) to set the tag to pull to a more recent version.

**Comment by KodrAus** [comment]
I've just published a new version of the Helm chart as `2023.2.0`. We ended up deciding that since the versioning scheme for the Helm chart is basically the same as Seq itself we should probably still push releases for it that line up with major/minor versions of Seq, even when the chart itself hasn't changed.

Thanks @ThomasErvik!
